package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@roadiehq/backstage-plugin-vouch-redirect",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "build-rollup": "rollup -c",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/catalog-model": "^0.2.0",
```

Comment:
I would suggest to bump plugins version to the latest ( catalog, cli etc.)

Reply:
All unneeded dependencies are yet to be excluded, rest of them has to be updated to the latest version

Reply:
Removed all unneeded dependencies, bumped cli dep